rasa/nlu/components.py

```diff
@@ -65,24 +64,58 @@ def validate_arguments(
         raise ValueError(
             "Can not train an empty pipeline. "
             "Make sure to specify a proper pipeline in "
-            "the configuration using the `pipeline` key."
-            + "The `backend` configuration key is "
+            "the configuration using the `pipeline` key. "
+            "The `backend` configuration key is "
             "NOT supported anymore."
         )
 
     provided_properties = set(context.keys())
 
     for component in pipeline:
         for r in component.requires:
-            if r not in provided_properties:
-                raise Exception(
-                    "Failed to validate at component "
-                    "'{}'. Missing property: '{}'"
-                    "".format(component.name, r)
-                )
+            if isinstance(r, Tuple):
+                validate_requires_any_of(r, provided_properties, str(component.name))
+            else:
+                if r not in provided_properties:
+                    raise Exception(
+                        f"Failed to validate component {component.name}. "
+                        f"Missing property: '{r}'"
+                    )
+
         provided_properties.update(component.provides)
 
 
+def any_of(*args):
```

Comment:
```suggestion
def any_of(*args) -> Tuple:
```
does this always return a tuple? not sure with *args, but would be helpful esp cause we're checking the type in L76